JS/Lesson31/onur/my-app/src/components/ListItem/index.jsx

```diff
@@ -1,9 +1,18 @@
+import { useState } from 'react';
 import './styles.css';
 
 export const ListItem = ({ id, title, handleDelete }) => {
+
+	const [completed, setCompleted] = useState(false);
+
+	const className = (!completed) ? '' : 'completed';
+
+	const changeClassName = () => {
+		return (className === '') ? setCompleted(true) : setCompleted(false);
+	}
 	return (
-		<li>
-			{title}
+		<li onClick={changeClassName}>
```

Comment:
I'd solve this like this instead (an I think it's the most ofter used pattern in React):
1. 
```suggestion
		<li onClick={() => setCompleted(prev => !prev)}>
```
2. You can now remove `changeClassName` function 
3. Update the span below as following:
`<span className={completed ? 'completed' : ''}>{title}</span>`
4. You can now also remove this line above: `const className = (!completed) ? '' : 'completed';`